### PR TITLE
test_closeClient method

### PR DESCRIPTION
**Pull request by winsvega**
https://github.com/ethereum/retesteth/issues/17

### CODE REVIEW

File: libweb3jsonrpc/test.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+{ "name": "test_closeClient", "params": [], "order": [], "returns": false},
```
Comment (pirapira) [inline libweb3jsonrpc/test.json]: I'm just curious; what does this "returns" field mean?

**Comment by gumb0** [comment]
Couldn't you just send a SIGTERM from retesteth?

File: libweb3jsonrpc/test.json (hunk 1 above)
Comment (winsvega) [reply libweb3jsonrpc/test.json]: I guess. false or true  means bool.  and "" means string. judgind on how other .json descriptions are made

**Comment by winsvega** [comment]
how ? 
pclose  wont send SIGTERM

**Comment by winsvega** [comment]
hm. in theory there should be a way to call kill $pid on unix system. but would it be a correct way to close the client ?

**Comment by gumb0** [comment]
Yes, sending `SIGTERM` or `SIGINT` should be a correct way, the client is responsible for handling it correctly.

But to send it you need to know PID and you have to be either the owner of the client process or a superuser

**Comment by pirapira** [comment]
@winsvega I guess, if `retesteth` doesn't kill `eth`, after running the test suite, there will be 400 `eth` processes left.  Is this correct?

**Comment by winsvega** [comment]
not really. just the number of threads retesteth runs the eth instances.

**Comment by winsvega** [comment]
found a way to send sigterm on unix

**Pull request closed without merging**